src/App.scss

```diff
@@ -1,38 +1,38 @@
-.App {
+.container {
   text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+  .container__title {
+    font-size: 100;
   }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+  .container__inputs {
+    display: flex;
+    justify-content: space-between;
+    padding: 3% 30%;
+    height: 17%;
+    input {
+      width: 40%;
+      font-size: 40px;
+      border: none;
+      outline: none;
+      border-bottom: 1px solid silver;
+      font-weight: 100;
+    }
   }
-  to {
-    transform: rotate(360deg);
+  .container__wrapper {
```

Comment:
Container == wrapper. You should use one of them

Reply:
+